docs/userGuide/syntax/pageLayouts.mbdf

```diff
@@ -95,3 +101,52 @@ afterSetup(() => {
 </frontmatter>
 ```
 </span>
+
+#### Using Expressive Layout Templates
+
+In the `page.md` file of your layouts, it should come with the following reserved variable:
+
+<box>
+  {{ mainContentBody }}
+</box>
+
+which injects the actual page content in every page. This allows you to build layouts in different ways.
```

Comment:
Reads better this way?

> The `page.md` file of a layout should come with the reserved variable `{{ MAIN_CONTENT_BODY }}` which injects the actual page content in every page. This allows you to build layouts in different ways.

Alternatively, can be written this way (i.e., avoid breaking sentences across boxes):
> The `page.md` file of a layout should come with the following reserved variable which injects the actual page content in every page. 
>```
>{{ MAIN_CONTENT_BODY }}
>```
>This allows you to build layouts in different ways.